test/java/seedu/addressbook/common/UtilsTest.java

```diff
@@ -60,6 +61,7 @@ public void elementsAreUnique() throws Exception {
         assertNotUnique(null, 1, new Integer(1));
         assertNotUnique(null, null);
         assertNotUnique(null, "a", "b", null);
+        assertNotUnique(null, "a", "a", "b", null);
```

Comment:
The activity was for you to write a test case such as such as isAnyNull{..}

Anyhow the tests were almost complete. Just keep in mind the naming convention for test methods (the first part is feature/method under test)
E.g. isAnyNull_emptyList_falseReturned
Part 2 and 3 of the test name can be removed to indicate an increased test scope